src/node-red/getting-started/building-node-red-plugin.md

```diff
@@ -0,0 +1,309 @@
+---
+eleventyNavigation:
+  key: Node-RED Plugin
+  order: 3
+  parent: Getting Started
+meta:
+  title: Building Node-RED Plugin
+  description: Learn what plugins are and how to build a Node-RED plugin from scratch with our comprehensive step-by-step guide.
+  keywords: node-red plugin, how to build node-red plugin, building node-red plugin
+---
+
+# {{ meta.title }}
+
+This guide aims to provide you with an understanding of what Node-RED plugins are and the differences between nodes and plugins. Additionally, it will walk you through the process of creating a Node-RED plugin from scratch. In this practical guide, we will develop a simple plugin that displays deployment information, including the total deployment count and the timestamp of the last deployment, in a newly added sidebar.
```

Comment:
This is a doc page, not a guide? Not sure if we need this sentence? Should it be removed?